ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/utils/GeneratorUtils.java

```diff
@@ -155,4 +159,67 @@ public static Arguments toArgument(final Ds3Param ds3Param) {
         final String paramType = ds3Param.getType().substring(ds3Param.getType().lastIndexOf(".") + 1);
         return new Arguments(paramType, ds3Param.getName());
     }
+
+    /**
+     * Determines if a given response code denotes a non-error response
+     */
+    public static boolean isNonErrorCode(final int responseCode) {
```

Comment:
I would also look at the rest of the functions added here since they seem like they'd be useful in the other code generators.